x-pack/plugins/ingest_manager/server/routes/package_config/handlers.ts

```diff
@@ -5,7 +5,7 @@
  */
 import { TypeOf } from '@kbn/config-schema';
 import Boom from 'boom';
-import { RequestHandler } from 'src/core/server';
+import { RequestHandler, SavedObjectsErrorHelpers } from '../../../../../../src/core/server';
```

Comment:
I added the relative path here because I was getting an error like "cannot find module 'src/core/server'" when I added the helpers